docs/standard/serialization/system-text-json-overview.md

```diff
@@ -30,10 +30,12 @@ The library also provides classes for working with an in-memory document object
 ## Additional resources
 
 * [How to use the library](system-text-json-how-to.md)
-* [Source code](https://github.com/dotnet/runtime/tree/master/src/libraries/System.Text.Json)
-* [API reference](xref:System.Text.Json)
-* [Roadmap](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Text.Json/roadmap/README.md)
-* GitHub issues in the dotnet/corefx repository
-  * [Discussion about the development of System.Text.Json](https://github.com/dotnet/corefx/issues/33115) <!-- TODO: Issues are still not moved to the new repo-->
-  * [All System.Text.Json issues](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-System.Text.Json)
-  * [System.Text.Json issues labeled json-functionality-doc](https://github.com/dotnet/runtime/labels/json-functionality-doc)
+* [How to migrate from Newtonsoft.Json](system-text-json-migrate-from-newtonsoft-how-to.md)
+* [How to write converters](system-text-json-converters-how-to.md)
+* [System.Text.Json source code](https://github.com/dotnet/runtime/tree/master/src/libraries/System.Text.Json)
+* [System.Text.Json API reference](xref:System.Text.Json)
+<!-- * [Roadmap](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Text.Json/roadmap/README.md)-->
+* GitHub issues in the dotnet/corefx and dotnet/runtime repositories
```

Comment:
>To me, this level of detail is unncessary. If anything, I would just point to a call to action saying, you can file issues here and only link to dotnet/runtime/issues.

I'll remove the issue links.  Regarding a link to create product issues, there is already one at the bottom of all docs pages.

> I don't think we should have dotnet/corefx/issues links. That repo is archived and the issues there will be bulk-moved to dotnet/runtime anyway.

Throughout the migrate doc I have direct links to specific issues for each missing feature, so people can check out current progress on the feature or discussion about that feature -- do you think those should be removed also? If we keep them, I would update the links when the issues are moved.

Reply:
> - do you think those should be removed also? If we keep them, I would update the links when the issues are moved.

This could be a significant on-going maintenance cost. Whenever an issue moves, gets fixed/closed, etc., the doc would have to be updated. Is there a way to structure the docs/links such that we don't have this on-going cost?

If not, I would question the value these links provide. Maybe, its not worth it?
If they were all located in a common area, we would know to update that particular list (only).

Reply:
Yes, maybe it's not worth the trouble.  I'll take them out now and after the issues are moved I could add back one link in the Additional resources section that lists all issues with the System.Text.Json label. That shouldn't require maintenance.